items-service/src/routes.js

```diff
@@ -0,0 +1,10 @@
+const { Router } = require('express');
+const ItemController = require('./controllers/ItemController');
+
+const routes = Router();
+
+routes.post('/items', ItemController.addItem);
+routes.post('/items/:id', ItemController.findItem);
```

Comment:
Rotas que são somente pra pegar dados, são do tipo GET

```suggestion
routes.get('/items/:id', ItemController.findItem);
```